geode-core/src/main/java/org/apache/geode/internal/cache/InitialImageOperation.java

```diff
@@ -1100,6 +1107,13 @@ protected Set<Object> processReceivedRVV(RegionVersionVector remoteRVV,
         }
       }
     }
+    if (!departedMemberSet.isEmpty() && foundIds.size() > 0) {
```

Comment:
instead of size() > 0 use !isEmpty()
also I don't see any need to check departedMemberSet here if you make the other changes.

Reply:
!departedMemberSet.isEmpty means this is non persistent region and there's something to remove to RVV's member map.